src/main/java/com/wisedevlife/whytalkuser/dto/request/MatchingRequest.java

```diff
@@ -0,0 +1,13 @@
+package com.wisedevlife.whytalkuser.dto.request;
+
+import com.wisedevlife.whytalkuser.common.dto.RangePair;
+import com.wisedevlife.whytalkuser.common.enums.SexEnum;
+import java.util.List;
+import lombok.Getter;
+
+@Getter
+public class MatchingRequest {
+    private String userId;
```

Comment:
Which naming convention do you think is better for our JSON properties, camel case or snake case? It seems we have to choose a convention for all of our request and response payloads. XD